src/ui/widgets/Webcam/webcam.tsx

```diff
@@ -18,20 +19,17 @@ const WebcamProps = {
 export const WebcamComponent = (
   props: InferWidgetProps<typeof WebcamProps>
 ): JSX.Element => {
-  const { width, height } = useWindowDimensions();
   // Create image tag
   return (
     <div
       style={{
-        height: height,
-        width: width,
         fontSize: "14px", // Set properties for image alt text
         fontWeight: "bold",
         color: "#a6190f"
       }}
     >
       <img
-        id="stream"
+        id={props.name}
```

Comment:
Is the use of name and id consistent with other widgets?

Reply:
Quite a few other widgets use the name prop e.g. GroupBox, usually in conjuction with the name attribute in the HTML element. The name attribute doesn't exist on the image element however, so I'm using id instead.